### PR TITLE
Add google-ads provider

### PR DESCRIPTION
**Pull request by TBonnin**
## Describe your changes

Adding google-ads provider

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "dependencies": {
         "@babel/traverse": "^7.22.5",
         "@inquirer/prompts": "^2.3.0",
-        "@nangohq/shared": "^0.36.101",
+        "@nangohq/shared": "^0.37.3",
```
Comment (TBonnin) [inline packages/cli/package.json:29]: unrelated to google-ads but I noticed that `shared` version had not been bumped by the publish script. I will investigate and try to understand why

**Review by khaliqgant — COMMENTED** [review]
Does the provider logo show up on the integrations page in this case? I think I had to symlink if the logo is different but in this case isn’t there a specific google ads logo? If so can you copy that svg? Also this provider should be added to the docs as well

**Comment by TBonnin** [comment]
logo doesn't show indeed. I will add the google ads one then.
Sorry I had forgotten to git add the doc file

**Comment by TBonnin** [comment]
@khaliqgant added the logo
Screenshot 2024-01-24 at 15 45 50

**Review by khaliqgant — APPROVED** [review]

**Pull request merged**